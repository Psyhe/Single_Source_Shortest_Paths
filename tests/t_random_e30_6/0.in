30 0 4
0 20 0
0 18 9
0 19 4
0 8 3
0 14 10
1 28 10
2 22 3
2 12 1
2 16 0
3 28 0
3 10 5
4 29 1
4 7 4
4 22 9
4 19 10
4 25 4
4 25 3
