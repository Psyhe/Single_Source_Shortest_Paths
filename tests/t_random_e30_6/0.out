0
30
15
20
13
